integration/util.go

```diff
@@ -143,7 +144,7 @@ func NewTempoQuerier() *e2e.HTTPService {
 
 	s := e2e.NewHTTPService(
 		"querier",
-		image,
+		queryImage,
```

Comment:
```suggestion
		image,
```


`querier != tempo-query`. Naming is confusing, but they're different components. The querier is a component in the read path, while tempo-query is an adapter to make Tempo work with Jaeger

Reply:
Fixed! thanks!